var/spack/repos/builtin/packages/universal-ctags/package.py

```diff
@@ -26,3 +26,6 @@ class UniversalCtags(AutotoolsPackage):
     depends_on('m4',       type='build')
     depends_on('libiconv', type='link')
     depends_on('pkgconfig', type='build')
+
+    def autoreconf(self, spec, prefix):
```

Comment:
Does this apply to *all* versions (in the package) or a specific version on?

Reply:
Good catch.  Looks like it was introduced in https://github.com/universal-ctags/ctags/commit/1d395ed94f9cb9b69204532ec11bdc318fb4c26a on Jun 2, 2014.  But the documentation and travis scripts aren't switched over until https://github.com/universal-ctags/ctags/commit/9d5b773c91e4afdb9792bdf2a5775df9a2522636#diff-7544c8642acdd15d295934770061473e64323316cd9797fb78c1830ebba08cf2 (Dec 2015)

a `git tag` shows the earliest tag as p5.9.20201011.0 which also contains the above commit.

Reply:
If all of the available tagged versions require the method then it sounds like you don't need to add an `@when('@<some-version>:')` directive to it (e.g., around line 100 of https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/dyninst/package.py).